.TAOS-CI/.gbs.conf

```diff
@@ -27,7 +27,7 @@ buildroot = ~/GBS-ROOT-Snapshot/
 url = https://api.tizen.org
 
 [repo.base]
-url = http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/latest/repos/standard/packages/
+url = http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/reference/repos/standard/packages/
 
 [repo.unified]
 url = http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/latest/repos/standard/packages/
```

Comment:
Please use `reference` instead of `latest`

Reply:
Updated!